src/main/java/net/openhft/chronicle/bytes/Bytes.java

```diff
@@ -207,10 +266,13 @@
      * }</pre>
      *
      * @param byteBuffer to wrap
-     * @return a Bytes which wraps the provided ByteBuffer and is ready for reading.
+     * @return a new Bytes wrapping the provided {@code byteBuffer}
+     * @throws NullPointerException    if the provided {@code byteBuffer} is {@code null}
+     * @throws ReadOnlyBufferException if the provided {@code byteBuffer} is read-only
```

Comment:
Should we be throwing a `ReadOnlyBufferException` when getting a read-only buffer? we're wrapping for read aren't we? it seems counter-intuitive that we require a writable buffer for this method

Reply:
I will raise a separate issue for this